tools/Customization/DevHome.Customization/Models/RepositoryInformation.cs

```diff
@@ -21,13 +20,22 @@ public class RepositoryInformation
 
     public string SourceControlProviderPackageDisplayName { get; }
 
-    public RepositoryInformation(string rootpath, string classId)
+    public string RepoPathMapping { get; }
```

Comment:
Please use `Repository` and not `Repo`.  I want DevHome to stay consistent in regards to that word.

Reply:
Modified in latest. Thanks!